apps/styleguide/src/tables/RowEditModal.vue

```diff
@@ -274,8 +273,13 @@ export default {
         let data = val[0];
         let defaultValue = {};
         this.tableMetadata.columns.forEach((column) => {
-          //skip key in case of clone
-          if (data[column.id] && (!this.clone || column.key != 1)) {
+          //skip key in case of clone and visible
+          if (
+            data[column.id] &&
+            (!this.clone ||
+              column.key != 1 ||
```

Comment:
why '1' ?

Reply:
1 = primary key. You want to empty that field on clone such that user needs to change it otherwise they get duplicate key exception